tests/test_cli.py

```diff
@@ -642,3 +678,19 @@ def test_run_non_universal_newline(self):
             '\n' % path)
         self.assertEqual(
             (ctx.returncode, ctx.stdout, ctx.stderr), (1, expected_out, ''))
+
+
+class CommandLineConfigTestCase(unittest.TestCase):
+    def test_yamllint_yml(self):
+        with temp_workspace({'.yamllint.yml': 'extends: relaxed\n'}):
+            with RunContext(self) as ctx:
+                cli.run(('-f', 'parsable', '.'))
+
+        self.assertEqual((ctx.returncode, ctx.stdout, ctx.stderr), (0, '', ''))
+
+    def test_yamllint_yaml(self):
+        with temp_workspace({'.yamllint.yaml': 'extends: relaxed\n'}):
+            with RunContext(self) as ctx:
+                cli.run(('-f', 'parsable', '.'))
+
+        self.assertEqual((ctx.returncode, ctx.stdout, ctx.stderr), (0, '', ''))
```

Comment:
These two tests verify that `.yamllint.yaml` and `.yamllint.yml` config files are recognized. I put them in `test_cli.py` since they're covering the following code which is otherwise untested:

https://github.com/adrienverge/yamllint/blob/e49a1011600d8e9831f205e659c389490c2cf7f5/yamllint/cli.py#L200

https://github.com/adrienverge/yamllint/blob/e49a1011600d8e9831f205e659c389490c2cf7f5/yamllint/cli.py#L202

I've rewritten them as a single test function to hopefully be more clear about what is being tested, and included `.yamllint` in the list of config files to verify. I also updated the test to assert that a warning is present in the absence of the config file.